<launch>

    <arg name="camera_name" default="phobos_nuclear"/>

<!-- STEREO REGISTRATOIN -->
    <arg name="localization" default="false"/>
    <include file="$(find i3dr_rtabmap)/launch/stereo_localisation.launch">
        <arg name="localization" value="$(arg localization)"/>
        <arg name="camera_namespace" value="$(arg camera_name)"/>
        <arg name="depth_odom" value="true"/>
        <!--<arg name="scan3d_odom" value="true"/>-->
        <!--<arg name="depth_icp_odom" value="true"/>-->
        <arg name="disparity2Depth" default="true"/>
        <arg name="rgb_topic" value="/$(arg camera_name)/left/image_rect_color"/>
        <arg name="camera_info_topic" value="/$(arg camera_name)/left/camera_info"/>
        <arg name="depth_camera_info_topic" value="/$(arg camera_name)/left/camera_info"/>
        <arg name="cloud_topic" value="/$(arg camera_name)/points2"/>
        <arg name="base_link_frame" value="$(arg camera_name)_base_link"/>
        <!-- should be set to true if using SGBM or Phobos/Phobos_nuclear as these takes longer to process -->
        <arg name="slow_processing" default="true"/>
    </include>

</launch>